src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -1214,10 +1244,6 @@ bool SlaterDetBuilder::readDetListH5(xmlNodePtr cur,
   std::string Dettype = "DETS";
   ValueType sumsq     = 0.0;
   OhmmsAttributeSet spoAttrib;
-  spoAttrib.add(NCA, "nca");
```

Comment:
Because you remove completely NCA, NEA NCB and NEB as well as the number of orbitals in the readDetListH5(), maybe it would be nice to have that information printed in the output (?). For now (and maybe because it is still in debug use) it is extra reassuring to see that I am having the correct states I am supposed to get...

Reply:
NCA, NEA NCB and NEB input are only kept for XML only case. They are removed in H5 case.
I will add a printing out in the h5 case.